docs/getting-started/04-expose-microservice.md

```diff
@@ -0,0 +1,188 @@
+
+</details>
+</div>
+
+### Call and test the microservice
+
+Follow these steps:
+
+> **CAUTION:** If you have a Minikube cluster, you must first add the IP address of the exposed k8s Service to the `hosts` file on your machine:
+>
+>  ```bash
+>  echo "$(minikube ip) orders-service.kyma.local" | sudo tee -a /etc/hosts
+>  ```
+
+1. Retrieve the domain of the exposed microservice and save it to the environment variable:
+
+  ```bash
+  export SERVICE_DOMAIN=$(kubectl get virtualservices -l apirule.gateway.kyma-project.io/v1alpha1=orders-service.orders-service -n orders-service -o=jsonpath='{.items[*].spec.hosts[0]}')
+  ```
+
+2. Run this command in the terminal to call the service:
+
+  ```bash
+  curl -ik "https://$SERVICE_DOMAIN/orders"
+  ```
+
+  The system returns a similar response:
+
+  ```bash
+  content-length: 2
+  content-type: application/json;charset=UTF-8
+  date: Mon, 13 Jul 2020 13:05:33 GMT
+  server: istio-envoy
+  vary: Origin
+  x-envoy-upstream-service-time: 37
+
+  []
+  ```
+
+3. Send a `POST` request to the microservice with sample order details:
+
+  ```bash
+  curl -ikX POST "https://$SERVICE_DOMAIN/orders" \
+    -H 'Cache-Control: no-cache' -d \
+    '{
+      "orderCode": "762727210",
+      "consignmentCode": "76272725",
+      "consignmentStatus": "PICKUP_COMPLETE"
+    }'
+  ```
+
+4. Call the microservice again to check the storage:
+
+  ```bash
+  curl -ik "https://$SERVICE_DOMAIN/orders"
+  ```
+
+  You should receive a similar response:
+
+  ```bash
+  HTTP/2 200
+  content-length: 73
+  content-type: application/json;charset=UTF-8
+  date: Mon, 13 Jul 2020 13:05:51 GMT
+  server: istio-envoy
+  vary: Origin
+  x-envoy-upstream-service-time: 6
+
+  [{"orderCode":"762727210","consignmentCode":"76272725","consignmentStatus":"PICKUP_COMPLETE"}]
+  ```
+
+  You can see the microservice returns the previously sent order details.
+
+5. Remove the [Pod](https://kubernetes.io/docs/concepts/workloads/pods/) created by the `orders-service` Deployment. Run this command and wait for the system to delete the Pod and start a new one:
+
+   ```bash
+   kubectl delete pod -n orders-service -l app=orders-service
+   ```
+
+6. Call the microservice again to check the storage:
+
+   ```bash
+   curl -ik "https://$SERVICE_DOMAIN/orders"
+   ```
+   The system returns a similar response:
+
+  ```bash
+  content-length: 2
+  content-type: application/json;charset=UTF-8
+  date: Mon, 13 Jul 2020 13:05:33 GMT
+  server: istio-envoy
+  vary: Origin
+  x-envoy-upstream-service-time: 37
+
+  []
+  ```
+
+  As you can see, the `orders-service` microservice uses in-memory storage which means every time you delete the Pod of the microservice or change the Deployment definition, the order details get lost. In further guides, you will see how you can prevent order data deletion by binding external Redis storage to the microservice.
```

Comment:
```suggestion
  As you can see, the `orders-service` microservice uses in-memory storage, which means every time you delete the Pod of the microservice or change the Deployment definition, you lose the order details. In further guides, you will see how you can prevent order data deletion by binding external Redis storage to the microservice.
```
1) The `which` used here is non-defining (it does not help you distinguish between different alternatives and only adds additional information non-essential for the distinction) and therefore needs a comma before it.
See the [reference](https://forum.wordreference.com/threads/comma-not-before-which-secondary-source-which-means.3155068/).
See the [reference](https://forum.wordreference.com/threads/comma-not-before-which-secondary-source-which-means.3155068/).

2) `the order details get lost` -> `you lose the order details` - this is just nitpicking. Feel free to ignore it and edit the suggestions, it's just `get lost` brings to my mind sad data that no-one wanted to play with and that was told to go away :D